### ISSUE TITLE
Pause essential trackers from notification

### ISSUE THREAD

**Issue by tfederico** [OxfordHCC/tracker-control-android#228]
Hi,

I was thinking it could be useful to add an option to enable the essential trackers from the persistent notification, in addition to enabling all the trackers for a certain amount of time.

**Comment by kasnder** [comment]
Hi there! Nice to see another PhD student around. :)

Thanks for the suggestion.

What apps are you thinking? Have you found many use cases, in which you had to allow essential trackers, and then could disabled them again?

**Comment by tfederico** [comment]
Usually I keep all the trackers deactivate by default, including the essential ones because some of them end up being not so essential. So when some website or app it's not working I use the notification menu to temporarily enable trackers, but it's a bit too permissive imho to enable all the trackers  :)

In addition, the Curve app won't open unless I enable trackers, same for YouTube music. Usually Spotify caches a certain number of songs when you start a playlist, so there should be no reason to keep the tracker active for a long period if the data is cached locally.

I also had a question: I noticed that a lot of trackers are mapped to US, is this GDPR compliant for European/UK users?

**Comment by kasnder** [comment]
> Usually I keep all the trackers deactivate by default, including the essential ones because some of them end up being not so essential. So when some website or app it's not working I use the notification menu to temporarily enable trackers, but it's a bit too permissive imho to enable all the trackers :)

Yeah! Unfortunately, Android puts limits on detecting the app in the foreground..

> In addition, the Curve app won't open unless I enable trackers, same for YouTube music. Usually Spotify caches a certain number of songs when you start a playlist, so there should be no reason to keep the tracker active for a long period if the data is cached locally

I see!

> I also had a question: I noticed that a lot of trackers are mapped to US, is this GDPR compliant for European/UK users?

I'm not a lawyer and cannot give legal advice. But I've got doubts about the Schrems II ruling, if such data contains personal information (which TC doesn't show you, but is often the case).

**Comment by kasnder** [comment]
Maybe, a workable improvement would be to only stop blocking ALL TRACKERS for a single app from the notification bar for some time?

**Comment by another-sapiens** [comment]
To add to this, maybe it would be useful to add a button that only allows the "Essencial" and "Uncategorized" trackers that made a request for the active app in, say, the last 10 seconds. 

In my experience, the uncategorized are usually the ones missing for the app to work. Most of the time, it shows a request "1 second ago" or something like that. When allowed, the app works again. 

Would it be a good workaround?
Could be a hit or miss, but at least the Analytics would still be blocked.
